tests_e2e/orchestrator/scripts/install-agent

```diff
@@ -51,9 +51,22 @@ if [ "$#" -ne 0 ] || [ -z ${package+x} ] || [ -z ${version+x} ]; then
 fi
 
 #
-# The service name is walinuxagent in Ubuntu and waagent elsewhere
+# Find the command to manage services
```

Comment:
Ubuntu has a 'service' wrapper on top of 'systemctl' but some of the distros I added don't, so our script needs to handle that difference.

Note that other distros that we will add do not use systemd, and services are managed with the 'service' command.